apps/catalogue/src/components/CohortCards.vue

```diff
@@ -103,20 +133,27 @@
 </template>
 
 <script>
-import { TableMixin, InputSearch } from "@mswertz/emx2-styleguide";
-
-import Property from "../components/Property";
-import ContributorList from "./ContributorList";
+import { InputSearch } from "molgenis-components";
+import { Client } from "molgenis-components";
 
 export default {
-  extends: TableMixin,
   components: {
-    Property,
```

Comment:
I am amazed I didn't spot these. Will double check my highlighting (recently switched back from IntelliJ dark to light mode because then coloration of warnings is shown better)